src/databricks/labs/blueprint/logger.py

```diff
@@ -54,8 +54,7 @@ def format(self, record: logging.LogRecord) -> str:
         level = self._levels[record.levelno]
         # databricks.labs.ucx.foo.bar -> d.l.u.foo.bar
         module_split = record.name.split(".")
-        last_two_modules = len(module_split) - 2
-        name = ".".join(part if i >= last_two_modules else part[0] for i, part in enumerate(module_split))
+        name = ".".join([*[c[:1] for c in module_split[:-2]], *module_split[-2:]])
```

Comment:
The last two modules were done to strip and provide a logging like below

for eg:
note for self: 
`databricks.labs.remorph.reconcile.execute` logs in the console as `d.l.r.reconcile.execute`

Reply:
@sundarshankar89: It still does that, but using slices instead.

I'll adjust to make it clearer how it works. I'm making use of the fact that unlike indexing, slices in Python don't trigger `IndexError()` with out-of-range values: the slice is just truncated.

Reply:
@sundarshankar89: I've updated this, is it a bit clearer now?